deploy/compose/docker-compose.yaml

```diff
@@ -259,6 +259,26 @@ services:
       - 'mongodb_data:/bitnami/mongodb'
     networks:
       - curiemesh
+     
+  update-filters-maxmind:
+    container_name: update-filters-maxmind
+    hostname: update-filters-maxmind
+    image: "gcr.io/rbz-dev-002/update-filters-maxmind:${DOCKER_TAG}"
+    restart: always
+    environment:
+      - CONFSERVER_SERVICE_HOST=confserver
+      - CONFSERVER_SERVICE_PORT=80
+      - SAASBACKEND_SERVICE_HOST=saasbackend
+      - SAASBACKEND_SERVICE_PORT=8911
+      - REMOTE_BUCKET=rbz-dev-auto-acl
+      - GOOGLE_APPLICATION_CREDENTIALS=/run/secrets/gcp_key
+    secrets:
+      - gcp_key
```

Comment:
This secret is not part of the Dockerfile. What is it for? How should users of curiefense set it? Having a GCP account does not seem to be an acceptable prerequisite for the open source curiefense project.